test/fixtures/registry-mocks/content/isaacs/testing-peer-dep-conflict-chain-dd.json

```diff
@@ -0,0 +1,3 @@
+{
+  "error": "Not found"
+}
```

Comment:
ah, I don't find any other ref to `testing-peer-dep-conflict-chain-dd` - is this intentional?

Reply:
Arg, it looks like I tried to fetch these before they switched over to `access: public` on the registry.  There seems to be a delay where they're private for a moment.  I'll update.

But yes, there's nothing using these at the moment.

Reply:
Oh, my mistake, no, those ended up living at ii, jj, ...

Removed.